airbyte-webapp/src/components/hooks/services/useDestinationHook.tsx

```diff
@@ -26,7 +26,9 @@ export const useDestinationDefinitionSpecificationLoad = (
   destinationDefinitionId: string
 ): {
   isLoading: boolean;
-  destinationDefinitionSpecification: DestinationDefinitionSpecification;
+  destinationDefinitionSpecification:
```

Comment:
I suppose it's better to declare it as `destinationDefinitionSpecification?` as this is the pattern we are trying to follow everywhere.